src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -67,6 +67,14 @@
                 localStorage.setItem(profilerStorageKey + name, value);
             },
 
+            addEvent = function (element, eventName, callback) {
+                if (document.all) {
```

Comment:
Maybe better to check the existence of `element.addEventListener`, don't you think?